common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2743,7 +2743,7 @@
   },
   "INVALID_INTERVAL_WITH_MICROSECONDS_ADDITION" : {
     "message" : [
-      "Cannot add an interval to a date because its microseconds part is not 0. If necessary set <ansiConfig> to \"false\" to bypass this error."
+      "Cannot add an interval to a date because its microseconds part is not 0."
```

Comment:
You could suggest explicitly convert the input date to a timestamp. In that case, the user can add the interval with non-zero microseconds.

Reply:
Ahh makes sense, thanks.

Let me add: "To resolve this, cast the input date to a timestamp, which supports the addition of intervals with non-zero microseconds."